### PR TITLE
Add method to get clients IP address

### PR DESCRIPTION
**Pull request by dsturm**
Add method to get clients IP address, even if forwarded; added filter to alter saved IP. Addresses #115

### CODE REVIEW

File: inc/class-session.php [hunk 1]
```diff
@@ -186,6 +186,39 @@ public function set_data( $data ) {
...
+			'HTTP_FORWARDED_FOR',
+			'HTTP_FORWARDED',
+			'REMOTE_ADDR',
+		];
+
+		foreach ( $keys as $key ) {
+			if ( array_key_exists( $key, $_SERVER )
+				&& $_SERVER[ $key ]
+			) {
+				$ipaddress = $_SERVER[$key];
```
Comment (danielbachhuber) [inline inc/class-session.php]:
```suggestion
				$ipaddress = $_SERVER[ $key ];
```

**Comment by danielbachhuber** [comment]
Sorry for the delay, @dsturm ! I landed your code in #122 and just tagged [v0.8.0](https://github.com/pantheon-systems/wp-native-php-sessions/releases/tag/v0.8.0)

**Comment by danielbachhuber** [comment]
And — thanks for the pull request 😄

**Comment by dsturm** [comment]
You're welcome 😊

**Pull request closed without merging**